### DISCUSSION TITLE
Dagster materialization of dbt seed fails with spaces in seed file column name

### DISCUSSION THREAD

**Discussion by benjaminifaught** [dagster-io/dagster#14685] [Q&A] [answered]
Hi! 

I'm just getting started with a Dagster POC on my team primarily to run dbt jobs. However, I am encountering some behavior that is different from what we see when executing dbt seed from the dbt cli. The issue is that our dbt project has some seeds with spaces in the column names. Even with my seed configuration in dbt_project.yml set with the "+quote_columns: true" the materialization fails in Dagster UNLESS I change the columns in the seed file to have underscores rather than spaces. 

This does not materialize:
CUSTOMER,HQ Flag
1234,TRUE
5678,TRUE

This does materialize:
CUSTOMER,HQ_Flag
1234,TRUE
5678,TRUE

Am I doing something wrong or is this intended behavior? 

Thanks!

**Comment by benjaminifaught** [comment, marked as answer]
False Alarm! I fixed it by adding column level quote configs for the seed files. Thanks!

**Comment by srishti-bhargava** [comment]
How were you runnning dbt seed from dagster?
